### ISSUE TITLE
Rocket League refuses to run on Ubuntu 20.04

### ISSUE THREAD

**Issue by keli5** [Heroic-Games-Launcher/HeroicGamesLauncher#315]
**Describe the bug**
When opening Rocket League, a window opens for a few seconds with a black screen and closes.

**To Reproduce**
1. Install Rocket League
2. Set to use Wine 6.6
3. Open game
4. Closes itself

**Expected behavior**
Rocket League opens and is playable

**Desktop (please complete the following information):**

- OS: Ubuntu
- Version: 20.04 Cinnamon
- Game: Rocket League
- Wine/Proton version: 6.6 Staging

**Additional context**
Game will not run in Proton 5.13, Proton 6.3, or Wine 6.6 staging, with or without DXVK, even in a clean wine prefix

**Comment by adityaruplaha** [comment]
Can you please post your log? Click on Settings > Latest Log.

**Comment by keli5** [comment]
Here's the most recent log in a clean Wine 6.6 prefix with install/update DXVK enabled:
`[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[Core] INFO: Getting authentication token...
[cli] INFO: Launching Sugar...
002c:fixme:winediag:LdrInitializeThunk wine-staging  is a testing version containing experimental patches.
002c:fixme:winediag:LdrInitializeThunk Please mention your exact version when filing bug reports on winehq.org.
Fontconfig error: Cannot load default config file
002c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
002c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Fontconfig error: Cannot load default config file
0058:err:winedevice:ServiceMain Failed to load L"C:\\windows\\system32\\win32k.sys"
Fontconfig error: Cannot load default config file
0034:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0034:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Fontconfig error: Cannot load default config file
0064:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0064:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0090:err:winedevice:ServiceMain Failed to load L"C:\\windows\\system32\\win32k.sys"
006c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
006c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Fontconfig error: Cannot load default config file
INTEL-MESA: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

008c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
008c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Fontconfig error: Cannot load default config file
00c8:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
00c8:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Fontconfig error: Cannot load default config file
0024:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0024:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Fontconfig error: Cannot load default config file
010c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
010c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
010c:fixme:toolhelp:CreateToolhelp32Snapshot Unimplemented: heap list snapshot
010c:fixme:toolhelp:Heap32ListFirst : stub
010c:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
010c:fixme:heap:RtlSetHeapInformation 0000000001320000 0 00000000004EE870 4 stub
info:  Game: RocketLeague.exe
info:  DXVK: v1.8.1
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
INTEL-MESA: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

info:  Intel(R) UHD Graphics 630 (CFL GT2):
info:    Driver: 20.2.6
info:    Vulkan: 1.2.145
info:    Memory Heap[0]: 
info:      Size: 11866 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0xf
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_0
info:  D3D11CoreCreateDevice: Using feature level D3D_FEATURE_LEVEL_11_0
info:  Device properties:
info:    Device name:     : Intel(R) UHD Graphics 630 (CFL GT2)
info:    Driver version   : 20.2.6
info:  Enabled device extensions:
info:    VK_EXT_4444_formats
info:    VK_EXT_custom_border_color
info:    VK_EXT_depth_clip_enable
info:    VK_EXT_extended_dynamic_state
info:    VK_EXT_host_query_reset
info:    VK_EXT_robustness2
info:    VK_EXT_shader_demote_to_helper_invocation
info:    VK_EXT_shader_stencil_export
info:    VK_EXT_shader_viewport_index_layer
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_create_renderpass2
info:    VK_KHR_depth_stencil_resolve
info:    VK_KHR_draw_indirect_count
info:    VK_KHR_driver_properties
info:    VK_KHR_image_format_list
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_shader_float_controls
info:    VK_KHR_swapchain
info:  Device features:
info:    robustBufferAccess                     : 1
info:    fullDrawIndexUint32                    : 1
info:    imageCubeArray                         : 1
info:    independentBlend                       : 1
info:    geometryShader                         : 1
info:    tessellationShader                     : 1
info:    sampleRateShading                      : 1
info:    dualSrcBlend                           : 1
info:    logicOp                                : 1
info:    multiDrawIndirect                      : 1
info:    drawIndirectFirstInstance              : 1
info:    depthClamp                             : 1
info:    depthBiasClamp                         : 1
info:    fillModeNonSolid                       : 1
info:    depthBounds                            : 0
info:    multiViewport                          : 1
info:    samplerAnisotropy                      : 1
info:    textureCompressionBC                   : 1
info:    occlusionQueryPrecise                  : 1
info:    pipelineStatisticsQuery                : 1
info:    vertexPipelineStoresAndAtomics         : 0
info:    fragmentStoresAndAtomics               : 1
info:    shaderImageGatherExtended              : 1
info:    shaderStorageImageExtendedFormats      : 1
info:    shaderStorageImageReadWithoutFormat    : 0
info:    shaderStorageImageWriteWithoutFormat   : 1
info:    shaderClipDistance                     : 1
info:    shaderCullDistance                     : 1
info:    shaderFloat64                          : 1
info:    shaderInt64                            : 1
info:    variableMultisampleRate                : 1
info:  VK_EXT_4444_formats
info:    formatA4R4G4B4                         : 1
info:    formatA4B4G4R4                         : 0
info:  VK_EXT_custom_border_color
info:    customBorderColors                     : 1
info:    customBorderColorWithoutFormat         : 1
info:  VK_EXT_depth_clip_enable
info:    depthClipEnable                        : 1
info:  VK_EXT_extended_dynamic_state
info:    extendedDynamicState                   : 1
info:  VK_EXT_host_query_reset
info:    hostQueryReset                         : 1
info:  VK_EXT_memory_priority
info:    memoryPriority                         : 0
info:  VK_EXT_robustness2
info:    robustBufferAccess2                    : 1
info:    robustImageAccess2                     : 1
info:    nullDescriptor                         : 1
info:  VK_EXT_shader_demote_to_helper_invocation
info:    shaderDemoteToHelperInvocation         : 1
info:  VK_EXT_transform_feedback
info:    transformFeedback                      : 1
info:    geometryStreams                        : 1
info:  VK_EXT_vertex_attribute_divisor
info:    vertexAttributeInstanceRateDivisor     : 1
info:    vertexAttributeInstanceRateZeroDivisor : 1
info:  Queue families:
info:    Graphics : 0
info:    Transfer : 0
info:  DXVK: Read 0 valid state cache entries
info:  DXVK: Using 3 compiler threads
info:  Game: RocketLeague.exe
info:  DXVK: v1.8.1
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
INTEL-MESA: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

info:  Intel(R) UHD Graphics 630 (CFL GT2):
info:    Driver: 20.2.6
info:    Vulkan: 1.2.145
info:    Memory Heap[0]: 
info:      Size: 11866 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0xf
010c:fixme:nvapi:unimplemented_stub function 0xbe7692ec is unimplemented!
010c:fixme:nvapi:unimplemented_stub function 0x6c2d048c is unimplemented!
010c:fixme:faultrep:ReportFault 00000000004EE2C0 0x0 stub`

**Comment by keli5** [comment]
A much shorter log in (another) clean x64 prefix, without DXVK:
```
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[Core] INFO: Getting authentication token...
[cli] INFO: Launching Sugar...
002c:fixme:winediag:LdrInitializeThunk wine-staging  is a testing version containing experimental patches.
002c:fixme:winediag:LdrInitializeThunk Please mention your exact version when filing bug reports on winehq.org.
Fontconfig error: Cannot load default config file
002c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
002c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Fontconfig error: Cannot load default config file
0058:err:winedevice:ServiceMain Failed to load L"C:\\windows\\system32\\win32k.sys"
Fontconfig error: Cannot load default config file
0034:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0034:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
005c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
005c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Fontconfig error: Cannot load default config file
Fontconfig error: Cannot load default config file
0064:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0064:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
006c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
006c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
INTEL-MESA: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

0098:err:winedevice:ServiceMain Failed to load L"C:\\windows\\system32\\win32k.sys"
Fontconfig error: Cannot load default config file
0094:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0094:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Fontconfig error: Cannot load default config file
0024:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0024:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Fontconfig error: Cannot load default config file
00fc:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
00fc:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
00fc:fixme:toolhelp:CreateToolhelp32Snapshot Unimplemented: heap list snapshot
00fc:fixme:toolhelp:Heap32ListFirst : stub
00fc:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
00fc:fixme:heap:RtlSetHeapInformation 0000000001320000 0 00000000004EE870 4 stub
00fc:fixme:dxgi:DXGID3D10CreateDevice Ignoring flags 0x1.
00fc:fixme:nvapi:unimplemented_stub function 0xbe7692ec is unimplemented!
00fc:fixme:nvapi:unimplemented_stub function 0x6c2d048c is unimplemented!
00fc:fixme:faultrep:ReportFault 00000000004EE2C0 0x0 stub
```

**Comment by adityaruplaha** [comment]
Oh. This is a game issue, not a Heroic or Legendary issue. You'll have to look for help at a Linux gaming forum.

Closing.
